implementation.md

```diff
@@ -419,8 +429,28 @@ is used to discover the properties of the action.
 
 For details on the parameters to an action, see below.
```

Comment:
This should probably refer to the section by name now, since 'below' is unrelated